content/en/network_monitoring/performance/setup.md

```diff
@@ -182,7 +182,7 @@ To enable Network Performance Monitoring for Windows hosts:
 
 1. Install the [Datadog Agent][1] (version 7.27.1 or above) with the network driver component enabled.
 
-   During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the agent installation through the GUI.
+   _(version 7.44 or below)_ During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the agent installation through the GUI.
```

Comment:
```suggestion
   _(version 7.44 or previous)_ During installation pass `ADDLOCAL="MainApplication,NPM"` to the `msiexec` command, or select "Network Performance Monitoring" when running the Agent installation through the GUI.
```

Reply:
Again it feels bit weird that we are still "advertising" the option. Perhaps we should have a separate table with "legacy" supported option?

Reply:
Thanks for mentioning that, it's a good point. Although in this specific example, this does seem like a relevant place to mention this difference, given that the line above it mentions versions 7.21.1 or above. This review suggests updating this to be a more eye-catching not, but I'm open to feedback and happy to keep iterating on this as needed.